[2024-01-19T09:29:18.153+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T09:29:17.442977+00:00 [queued]>
[2024-01-19T09:29:18.157+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T09:29:17.442977+00:00 [queued]>
[2024-01-19T09:29:18.157+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-19T09:29:18.162+0000] {taskinstance.py:2192} INFO - Executing <Task(BranchPythonOperator): CheckTableExisting> on 2024-01-19 09:29:17.442977+00:00
[2024-01-19T09:29:18.166+0000] {standard_task_runner.py:60} INFO - Started process 142 to run task
[2024-01-19T09:29:18.169+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'company_profile_pipeline', 'CheckTableExisting', 'manual__2024-01-19T09:29:17.442977+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/CrawlCompanyInfo.py', '--cfg-path', '/tmp/tmpsvggcjym']
[2024-01-19T09:29:18.169+0000] {standard_task_runner.py:88} INFO - Job 80: Subtask CheckTableExisting
[2024-01-19T09:29:18.190+0000] {task_command.py:423} INFO - Running <TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T09:29:17.442977+00:00 [running]> on host 56f123d3e42c
[2024-01-19T09:29:18.222+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='anh.ho@tititada.com' AIRFLOW_CTX_DAG_OWNER='anh.ho' AIRFLOW_CTX_DAG_ID='company_profile_pipeline' AIRFLOW_CTX_TASK_ID='CheckTableExisting' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T09:29:17.442977+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-19T09:29:17.442977+00:00'
[2024-01-19T09:29:18.223+0000] {logging_mixin.py:188} INFO - Connecting to the PostgreSQL database...
[2024-01-19T09:29:18.225+0000] {logging_mixin.py:188} INFO - PostgreSQL database version:
[2024-01-19T09:29:18.225+0000] {logging_mixin.py:188} INFO - Successfully connect to Postgresql
[2024-01-19T09:29:18.225+0000] {logging_mixin.py:188} INFO - ('PostgreSQL 13.13 (Debian 13.13-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit',)
[2024-01-19T09:29:18.225+0000] {logging_mixin.py:188} INFO - Checking existing table...
[2024-01-19T09:29:18.227+0000] {python.py:201} INFO - Done. Returned value was: ExistingCompanyProfile
[2024-01-19T09:29:18.227+0000] {branch.py:35} INFO - Branch into ExistingCompanyProfile
[2024-01-19T09:29:18.227+0000] {skipmixin.py:178} INFO - Following branch ExistingCompanyProfile
[2024-01-19T09:29:18.233+0000] {skipmixin.py:244} INFO - Skipping tasks [('CreateCompanyProfileTable', -1)]
[2024-01-19T09:29:18.249+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=company_profile_pipeline, task_id=CheckTableExisting, execution_date=20240119T092917, start_date=20240119T092918, end_date=20240119T092918
[2024-01-19T09:29:18.263+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-19T09:29:18.279+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
